### PR TITLE
Hongyli new case aug17 veri

### PR DESCRIPTION
**Pull request by lihongyan1**
Related to PR:
https://github.com/openshift/cucushift/pull/8043

### CODE REVIEW

**Comment by lihongyan1** [comment]
Please help review and merge

File: lib/openshift/flakes/container_spec.rb [hunk 1]
```diff
@@ -22,6 +22,10 @@ def image
         return struct['image']
       end
 
+      def args
```
Comment (liangxia) [inline lib/openshift/flakes/container_spec.rb:25]: The functions under the module are sorted.

Comment (pruan-rht) [reply lib/openshift/flakes/container_spec.rb:25]: @liangxia that convention is not really followed in practice at all...we do encourage the sorting to be done alphabetically in the rules file.  I'll merge this to unblock @lihongyan1

Comment (liangxia) [reply lib/openshift/flakes/container_spec.rb:25]: @pruan-rht  OK. Got it.

**Pull request merged**